packages/smooth_app/lib/pages/prices/prices_proofs_page.dart

```diff
@@ -192,9 +210,9 @@ class _PricesProofsPageState extends State<PricesProofsPage>
   }
 }
 
-// TODO(monsieurtanuki): reuse whatever will be coded in https://github.com/openfoodfacts/smooth-app/pull/5366
-class _PriceProofImage extends StatelessWidget {
-  const _PriceProofImage(
+/// Created a simplified version of _PriceProofImage if needed for transition
+class PriceProofImage extends StatelessWidget {
+  const PriceProofImage(
```

Comment:
Just trying to understand: you're creating dead code, and you want it to be public instead of private, right?